### DISCUSSION TITLE
Proxy settings

### DISCUSSION THREAD

**Discussion by zxcOvosh** [seerr-team/seerr#1010] [Ideas]
I hope for the earliest possible addition of the ability to connect a proxy socks5 through the application settings, as it is done in the radarr and sonarr

**Comment by fallenbagel** [comment]
Duplicate of #491

**Comment by zxcOvosh** [comment]
it would still be nice to make proxy support right in the interface like in radarr/sonarr

**Reply by fallenbagel** [reply]
Thats what the issue states too?
